native_client/python/client.py

```diff
@@ -17,10 +17,13 @@
 except ImportError:
     from pipes import quote
 
+# Define the sample rate for audio
+
+SAMPLE_RATE = 16000
 # These constants control the beam search decoder
 
 # Beam width used in the CTC decoder when building candidate transcriptions
-BEAM_WIDTH = 500
+BEAM_WIDTH = 1024
```

Comment:
Please remove this change, we default to a smaller beam width on the clients as they're optimized for low-latency local inference and the accuracy improvements beyond 500 aren't big.